### DISCUSSION TITLE
Add multiple routes to route().current()

### DISCUSSION THREAD

**Discussion by rotogro** [tighten/ziggy#510] [Ideas]
#### Description

It would be nice to be able to check multiple routes in <code>route().current()</code>. 
I sometimes have a nav hierarchy and would like the top node to be "active" when a child is active.
#### Suggestion

<code>route().current(['page1', 'page2', 'page3']);</code>

**Comment by bakerkretzmar** [comment]
Depending on your route names you may already be able to do this. `current` accepts parameters and some basic wildcard patterns, so for example if you have a route like `{parent}/{child}/{grandchild}` you could do something like `route().current('page.*', { parent: 'section1' })`. There a bunch more examples in [the tests](https://github.com/tighten/ziggy/blob/main/tests/js/route.test.js#L847).

**Reply by rotogro** [reply]
Yep.  The wildcard works for my routing structure. :)

**Comment by justrustyspoons** [comment]
Is it possible to do this with routes with different prefixes? In Laravel you can do something like this ```Route::is('hangar.*', 'aircraft.*')``` how can I achieve that with ziggy?
